### PR TITLE
Dont ensure running, #220

### PR DESCRIPTION
**Pull request by zramsay**

### CODE REVIEW

File: files/files_test.go [hunk 1]
```diff
@@ -122,6 +123,12 @@ func testsInit() error {
 	ifExit(err)
 	f.Write([]byte(content))
 
+	do1 := definitions.NowDo()
+	do1.Args = []string{"ipfs"}
+	err = services.StartService(do1)
+	time.Sleep(5 * time.Second)
```
Comment (ebuchman) [inline files/files_test.go]: sleep after checking the error :)

File: perform/docker_run.go [hunk 2]
```diff
@@ -265,14 +265,14 @@ func DockerRun(srv *def.Service, ops *def.Operation) (string, error) {
...
-			return "", err
+			return err
 		}
 
 	} else {
 		logger.Infof("Successfully started service =>\t%s\n", srv.Name)
 	}
 
-	return id_main, nil
+	return nil
```
Comment (ebuchman) [inline perform/docker_run.go]: is there some reason not to return this? presumably it was only used by ensure running? i can see it being useful for other things potentially

File: services/load.go [hunk 3]
```diff
@@ -22,44 +20,11 @@ func EnsureRunning(do *definitions.Do) error {
...
-					time.Sleep(500 * time.Millisecond)
-				}
-			}
-			time.Sleep(500 * time.Millisecond)
-			break MAIN_LOOP
-		}
+		errmsg := fmt.Sprintf("%s is not running, start it with `eris services start %s`\n", strings.ToUpper(do.Name), do.Name)
+		err := errors.New(errmsg)
+		return err
+		os.Exit(1)
```
Comment (ebuchman) [inline services/load.go]: can't have this. remember these functions need to work from an http server as well. you should define an error type called `ErrServiceNotRunning` and return that. 

actually, you're returning first so the Exit shouldn't trigger, but still, remove it and use a defined error.

Comment (ebuchman) [inline services/load.go]: Also fix the comments above EnsureRunning

File: perform/docker_run.go (hunk 2 above)
Comment (zramsay) [inline perform/docker_run.go]: it never did return it until I needed the container ID to get the exposed ports for ping (which this PR is eliminating). it also made `DockerRun()` ugly

Comment (zramsay) [inline perform/docker_run.go]: we can revisit later if need be...

Comment (compleatang) [inline perform/docker_run.go]: think cleaner without the return (for now). when we get to agents and remotes we may want to get this info as a return value but let's get a better grasp on things at that time.

**Comment by compleatang** [comment]
OK. This LGTM. @ebuchman? @zramsay you ready for me to merge?

**Pull request merged**